starter_code/index.html

```diff
@@ -5,22 +5,73 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
+		<title>Relaxr blog</title>
 
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
+		<link rel="stylesheet" href="css/index.css">
 
 	</head>
 	<body>
-		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+		<nav>
+			<header id="nav-bar">
+				<h1>Relaxr</h1>
+				<div id="nav-cards">
+					<p>About</p>
+					<p>FAQ</p>
+					<p>Team</p>
+					<p>Contact Us</p>
+					<p>Blog</p>
+				</div>
 			</header>
-			<footer>
-				&copy; Footer Goes Here
-			</footer>
-		</div><!-- #container -->
+		</nav>
+		<div class="container">
+			<div class="left">
+				<div class="article">
+					<h2>How I implemented Relaxr in 2 weeks and changed my life.</h2>
+					<img src="images/blog_photo1.jpg" alt="blog-photo1">
+					<p class="mock">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+					<p class="read-more"> Read more></p>
```

Comment:
using the <p> tag is fine, but I would have used an anchor tag because "read more" is meant to send you to another page.

Reply:
@ndChang 
see above comment